.github/workflows/conformance.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
+        python: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
Are we absolutely sure we should remove these two versions? Seems to me that we still support 3.7 and 3.8 according to https://screenshot.googleplex.com/4TSpCmAE3z63Axt https://screenshot.googleplex.com/4oTVpkPBHbbBw8b